django_slack_oauth/views.py

```diff
@@ -1,43 +1,46 @@
 # -*- coding: utf-8 -*-
 
 import uuid
+from importlib import import_module
 
-try:
-    from urllib.parse import urlencode
-except ImportError:
-    from urllib import urlencode
+import requests
 
 from django.contrib import messages
-from django.contrib.auth.decorators import login_required
 from django.core.urlresolvers import reverse
 from django.core.cache import cache
-from django.http.response import HttpResponseRedirect
-from django.utils.decorators import method_decorator
+from django.http.response import HttpResponseRedirect, HttpResponse
+from django.views import View
 from django.views.generic import RedirectView
 
-import requests
-
 from . import settings
-from .models import SlackUser
+
+try:
+    from urllib.parse import urlencode
+except ImportError:
+    from urllib import urlencode
 
 __all__ = (
     'SlackAuthView',
+    'DefaultSuccessView'
 )
 
 
 class StateMismatch(Exception):
     pass
 
 
+class DefaultSuccessView(View):
+
+    def get(self, request):
+        messages.success(request, "You've been successfully authenticated.")
+        return HttpResponse("Slack OAuth login successful.")
```

Comment:
It's wrong UX anyway to add an error and redirect to a success URL? Perhaps it would be better if we defined a `ERROR_REDIRECT_URL` too, in the same way?